stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/common/EmptyView.kt

```diff
@@ -39,7 +39,7 @@ public fun EmptyView(modifier: Modifier = Modifier) {
         Spacer(Modifier.size(16.dp))
 
         Text(
-            text = stringResource(id = R.string.stream_compose_empty_state_title),
+            text = stringResource(id = R.string.stream_compose_channel_list_empty_title),
             style = ChatTheme.typography.bodyBold,
```

Comment:
As discussed, this will be improved later. We need a slightly different wording for this `EmptyView` depending on the place where it is used.

Reply:
Yeah do make sure to check with Design if we want completely different UI for these placeholders or not. If we do - we can split them up in UI, otherwise we can just expose string/image params for this component!